extensions.md

```diff
@@ -14,36 +15,39 @@ We try to do this sparingly. The most important considerations are:
 These extensions may evolve or disappear over time. If you use them, try to
 recover gracefully if the structures aren't what's expected.
 
-{% include toc.md %} 
+{% include toc.md %}
 
 ## Switch between source/header
 {:.v6}
 
 Lets editors switch between the main source file (`*.cpp`) and header (`*.h`).
 
 **New client->server request**: `textDocument/switchSourceHeader`.
-  - Params: `TextDocumentIdentifier`: an open file.
```

Comment:
Hmm, this indented formatting was deliberate - it closely matches the rendering, and visually associates the params/result with the request as opposed to the overall heading.

Is there a reason it needs to be changed?

Reply:
From Markdown point of view this is just equivalent and the indentation usually means actual nested indentation (hence, idiomatic MD). Decreasing indentation level by 1 also saves some space.

I don't have a strong opinion, though, and am OK to revert this, this is simply surprising to me from the aesthetic point of view.